setup.py

```diff
@@ -39,14 +38,26 @@
 NO_PYTHON_ERROR = """
 WARNING: No #!python executable found in %s, skipping .bat wrapper'"""
 
+SCRIPT_ALIASES = {'/scripts/bibliopixel': '/scripts/bp'}
+
 
 class InstallScripts(install_scripts):
     def run(self):
-        install_scripts.run(self)
+        super().run()
+
+        extras = []
+        for filepath in self.get_outputs():
+            for alias, original in SCRIPT_ALIASES.items():
+                if filepath.endswith(original):
```

Comment:
It's the `endswith` that's bogus.  If I had a correct "install_base" I could simply check the URL relative to that, but I am not given that.